02-spatial-data.Rmd

```diff
@@ -652,10 +660,11 @@ extent(new_raster)
 crs(new_raster)
 ```
 
-Unlike the **sf** package, **raster** only use the `proj4string` representation of the coordinate reference system.
+<!--CRSargs(CRS("+init=epsg:4326"))-->
+Note that in contrast to the **sf** package, **raster** only accepts the `proj4string` representation of the coordinate reference system.
 
 Values of the cells are the only non-spatial information in the `Raster*` object.
```

Comment:
actually, I was referring to the raster cell values, but no worries, I've changed that.